File: src/Entities/Medicine.Entities/Models/CourseGroup.cs
using Medicine.Entities.Models.Translated;
using Medicine.Entities.Models.Translated.Base;

namespace Medicine.Entities.Models
{
    public class CourseGroup : TransatedEntityWithDescription
    {
        public List<Course> Courses { get; set; } = new List<Course>();
        public List<TranslatedCourseGroup> Translations { get; set; } = new List<TranslatedCourseGroup>();
    }
}

File: front/medicine/src/app/shared/models/entity/CourseGroup.ts
import { Language } from "../../enums/Language";
import { User } from "./User";
import { Course } from "./Course";
import { TranslatedCourseGroup } from "./translate/TranslatedCourseGroup";


export class CourseGroup {
  courses: Course[] = [];
  translations: TranslatedCourseGroup[] = [];
  title?: string;
  description?: string;
  language!: Language;
  user?: User;
  userId?: number;
  id!: number;
  createdAt?: Date;
}
